libs/service-portal/health/src/lib/navigation.ts

```diff
@@ -115,31 +141,39 @@ export const healthNavigation: PortalNavigationItem = {
     },
     {
       name: messages.vaccinations,
+      intro: messages.vaccinationsIntro,
+      serviceProvider: 'landlaeknir',
```

Comment:
Use constants for these

libs/service-portal/core/src/utils/constants.ts

Reply:
I've been wondering, why use the constants for this?
The service provider type is already a typed literal (`organizationSlugType`) , feels unnecessary to use an intermediary `const`

Reply:
Same string value being used in various places was the initital thought behind these. But yeah I agree since this has been typed we dont need it anymore 🥳